recipe/meta.yaml

```diff
@@ -22,8 +22,9 @@ requirements:
 test:
   commands:
     - makeinfo -h
-    #- conda inspect linkages -p $PREFIX texinfo  # [not win]
-    #- conda inspect objects -p $PREFIX texinfo  # [osx]
+    - conda inspect linkages -p $PREFIX texinfo  # [not win]
+    - conda inspect objects -p $PREFIX texinfo  # [osx]
+    - conda inspect prefix-lengths /feedstock_root/build_artefacts/linux-64/*.tar.bz2  # [linux]
```

Comment:
This is temporary because it won't be nice to use hard-coded paths in the recipes. But it would be nice if we could inspect the prefix length of a certain package with `-p $PREFIX texinfo` and make this permanent.